pkgs/development/tools/database/prisma-engines/default.nix

```diff
@@ -45,7 +43,7 @@ rustPlatform.buildRustPackage rec {
   '';
 
   postInstall = ''
-    cp target/release/libquery_engine.so $out/lib/libquery_engine.so.node
+    cp target/x86_64-unknown-linux-gnu/release/libquery_engine.so $out/lib/libquery_engine.so.node
```

Comment:
Also I would need to test but it seems like rust adds a lib folder by default. It contains `lib/libquery_engine.so` maybe that could also be used.